roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Stat for Contrail docker images
+  stat:
+    path: "/tmp/{{ item }}-{{ contrail_os_release }}-{{ contrail_version }}.tar.gz"
+  register: image
+  with_items:
+  - contrail-agent
+  - contrail-kubernetes-agent
+
+- name: Stat for loopback plugin
```

Comment:
Can't the daemonset image have the loopback pre-installed in it?

Reply:
Hi @rajatchopra we don't package that & pull from upstream